core/modules/simpletest/backdrop_web_test_case.php

```diff
@@ -1876,8 +1876,14 @@ protected function tearDown() {
       $this->fail('Failed to drop all prefixed tables.');
     }
 
+    // In PHP 8 some tests encounter problems as some shutdown code tries to
+    // access the the database connection after it's been explicitly closed, for
+    // example the destructor of DrupalCacheArray. We avoid this by not fully
+    // destroying the test database connection.
```

Comment:
```suggestion
    // In PHP 8 some tests encounter problems as some shutdown code tries to
    // access the database connection after it's been explicitly closed (for
    // example the destructor of DrupalCacheArray). We avoid this by not fully
    // destroying the test database connection.
```

Reply:
Thanks, this was a direct copy/paste from the associated Drupal issue. I'll include your changes in a follow-up commit.